docs/manual/creating-a-checker.tex

```diff
@@ -1075,14 +1075,17 @@
 
 \noindent
 if you define an aggregate checker class.  Extend \refclass{framework/source}{AggregateChecker} and override
-the \<getImmediateSubcheckerClasses> method, like the following:
+the \<getSupportedCheckers> method, like the following:
 
 \begin{Verbatim}
   public class MyUnitCheckers extends AggregateChecker {
-    protected Set<Class<? extends SourceChecker>> getImmediateSubcheckerClasses() {
```

Comment:
I think this might have been a copy-and-paste error. There are no instances in the CF where an aggregate checker overrides `getImmediateSubcheckerClasses`